src/main/java/uk/gov/hmcts/reform/blobrouter/services/NewEnvelopesFinder.java

```diff
@@ -74,4 +92,9 @@ private void checkNewEnvelopesInContainers(Set<String> containers, String contai
             .collect(Collectors.toSet());
     }
 
+    private boolean isCurrentTimeInBusinessHours(Clock clock) {
+        LocalDateTime now = LocalDateTime.now(clock);
+        return !WEEKEND.contains(now.getDayOfWeek()) // not weekend
+            && now.getHour() > START_HOUR && now.getHour() < END_HOUR; // between 10am-6pm
```

Comment:
if it's *start* hour, I'd expect `>=` (same for end)

Reply:
Updated the values and condition